bikram.md

```diff
@@ -0,0 +1,95 @@
+1. API Authentication:
+Every registered RESTful API should be assigned a unique Client ID and a Client Secret as a part of an HTTP header. The Client Secret should not be shared. Do not solely rely on a 'Client ID' for authentication.
+
+2	Access Control:
+Not every user has a right to every web service. This is vital, as you don't want administrative web services to be misused. The API key should be sent along as a cookie or body parameter to ensure that privileged collections or actions are properly protected from unauthorized use. Every API needs to be authenticated before it can be used.
```

Comment:
It's always better to provide such API keys as a form of body parameter. On a traffic governed by TLS, it would be difficult to actually get a hold of it.